0.35.0 release notes (#3801)
astronomer/docs: software/release-notes.md

:::

## 0.35.0 

> Release date: May 14, 2024

Reviewer: ```suggestion
Release date: June 14, 2024
```

Reviewer 2: We are not releasing it on June 14. Probably second last or last week of june.

### Use deploy rollbacks to run previous versions of your code

Deploy rollbacks are a new way to ensure that your Deployments can keep running after a broken code deploy or upgrade. When you trigger a rollback to a past deploy, your previous image and/or DAG code are redeployed and start running. 

> To configure deploy rollbacks, see [LIKNK]

Reviewer: to add after merge to release branch

### Support for MySQL

You can now configure Astronomer Software to use a MySQL database as the backend for both Houston and your Airflow Deployments. To connect your Astronomer Software installation to a MySQL database, you update the `astronomer-bootstrap` secret in your cluster to point to your database:

```sh
kubectl create secret generic astronomer-bootstrap \
  --from-literal connection="jdbc:mysql://host1:33060/mydb" \
  --namespace astronomer
```

### Create Teams without using an IdP

> Astronomer [Teams](import-idp-groups.md) can now be created without associating the team to an existing identity provider (IdP) group. Use local Teams to efficiently manage permissions for users with only a local username and password. See [LINK] for configuration steps.

Reviewer: adding link after merge into base branch

Reviewer 3: I would change this

From

"Astronomer [Teams](import-idp-groups.md) can now be created without associating the team to an existing identity provider (IdP) group. Use local Teams to efficiently manage permissions for users with only a local username and password. See [LINK] for configuration steps."

to

"Astronomer [Teams](import-idp-groups.md) can now be created without associating the team to an existing identity provider (IdP) group. Use local Teams to efficiently manage permissions for groups that might not exist in your identity provider. See [LINK] for configuration steps."

:::

## 0.35.0

Release date: June 28, 2024

### Use deploy rollbacks to run previous versions of your code

Deploy rollbacks are a new way to ensure that your Deployments can keep running after a broken code deploy or upgrade. When you trigger a rollback to a past deploy, your previous image and/or DAG code are redeployed and start running.

To configure deploy rollbacks, see [LIKNK]

### Support for MySQL

You can now configure Astronomer Software to use a MySQL database as the backend for both Houston and your Airflow Deployments. To connect your Astronomer Software installation to a MySQL database, you update the `astronomer-bootstrap` secret in your cluster to point to your database:

```sh
kubectl create secret generic astronomer-bootstrap \
  --from-literal connection="jdbc:mysql://host1:33060/mydb" \
  --namespace astronomer
```

### Create Teams without using an IdP

> Astronomer [Teams](import-idp-groups.md) can now be created without associating the team to an existing identity provider (IdP) group. Use local Teams to efficiently manage permissions for users with only a local username and password. See [LINK] for configuration steps.

Reviewer 4: Updating comment as a suggestion
```suggestion
Astronomer [Teams](import-idp-groups.md) ccan now be created without associating the team to an existing identity provider (IdP) group. Use local Teams to efficiently manage permissions for groups that might not exist in your identity provider. See [LINK] for configuration steps.
```
```suggestion
Astronomer [Teams](import-idp-groups.md) can now be created without associating the team to an existing identity provider (IdP) group. Use local Teams to efficiently manage permissions for users with only a local username and password. See [LINK] for configuration steps.
```

> Astronomer [Teams](import-idp-groups.md) can now be created without associating the team to an existing identity provider (IdP) group. Use local Teams to efficiently manage permissions for users with only a local username and password. See [LINK] for configuration steps.

Reviewer 4: Updating comment as a suggestion

```suggestion
Astronomer [Teams](import-idp-groups.md) can now be created without associating the team to an existing identity provider (IdP) group. Use local Teams to efficiently manage permissions for groups that might not exist in your identity provider. See [LINK] for configuration steps.
```

### Additional improvements

- DAG-only deploys are now fully compatible with OpenShift clusters.

> - You can now set a `priorityClass` to pods created by `templates/trust-private-ca-on-all-nodes/containerd-daemonset.yaml`. This allows you to set a high priority class for certain pods to force an evication for important processes. <!--https://github.com/astronomer/issues/issues/6366-->
> 

Reviewer: ```suggestion
- You can now set a `priorityClass` to pods created by `templates/trust-private-ca-on-all-nodes/containerd-daemonset.yaml`. This allows you to set a high priority class for certain pods to force an evication for important processes. <!--https://github.com/astronomer/issues/issues/6366-->
```
